sp800-63b/sec3_definitions.md

```diff
@@ -9,7 +9,7 @@ There is a variety of terms used in the area of authentication. While the defini
 The definitions in this section are primarily those that are referenced in this document. Refer to the other documents in the SP 800-63 document family for additional definitions and abbreviations specific to their content.
 
 #### Active Attack
-An attack on the authentication protocol where the attacker transmits data to the claimant, Credential Service Provider, verifier, or Relying Party. Examples of active attacks include man-in-the-middle, impersonation, and session hijacking.
+An attack on the authentication protocol where the attacker transmits data to the claimant, Credential Service Provider (CSP), verifier, or RP. Examples of active attacks include man-in-the-middle (MitM), impersonation, and session hijacking.
```

Comment:
The general pattern I discussed with Paul that I'm following is that if the acronym was introduced in an earlier section (in this case the intro for RP) we go with the acronym from that point forward in the rest of document sections. I don't have a problem with spelling out RP here though if it will help the definition stand on its own.